composer.json

```diff
@@ -19,7 +19,7 @@
   "require": {
     "php": "^8.1",
     "ext-json": "*",
-    "livewire/livewire": "^2.11.2",
+    "livewire/livewire": "^3.0.1",
```

Comment:
Maybe we should retain 2.x for backward compatibility? Thanks!

```suggestion
    "livewire/livewire": "^2.11.2|^3.0.1",
```

Reply:
Updated the PR. thank you!